src/POM/HomePage.js

```diff
@@ -0,0 +1,27 @@
+const {By} = require('selenium-webdriver');
+require('dotenv').config();
+
+class HomePage {
+	constructor(driver){
+		this.driver = driver;
+	}
+
+	async mainPage(){
+		await this.driver.get(process.env.HOMEPAGE_URL_LOCAL);
+	}
+
+
+	async openLoginPage(){
+		await this.driver.get(process.env.LOGIN_PAGE_URL_LOCAL);
```

Comment:
Any new environment variables need to be added to the .env example file

Reply:
done